src/core/unit.h

```diff
@@ -114,6 +114,7 @@ struct Unit {
 
         /* Job timeout and action to take */
         usec_t job_timeout;
+        usec_t job_running_timeout;
```

Comment:
hmm, i don't really follow why you don't need two timer events here, for the two different timeouts? can you elaborate?